library/ssl_tls.c

```diff
@@ -151,7 +151,7 @@ static int ssl_get_remaining_payload_in_datagram( mbedtls_ssl_context const *ssl
 {
     int ret;
     size_t remaining, expansion;
-    size_t max_len = MBEDTLS_SSL_MAX_CONTENT_LEN;
+    size_t max_len = MBEDTLS_SSL_OUT_CONTENT_LEN;
```

Comment:
I'm not really sure if this should belong here. It's a bugfix that's already handled by a separate PR on the mbedTLS repository. If we're in a hurry to fix this then sure, it's ok, but if not (and that's what I think because of the PR title), maybe we should just wait for it to be fixed in mainline mbedTLS?

Reply:
@AndrzejKurek Ok, I reverted the commit.